python/lsst/validate/drp/validate.py

```diff
@@ -88,17 +92,23 @@ def loadAndMatchData(repo, visitDataIds,
     srcVis = SourceCatalog(newSchema)
 
     for vId in visitDataIds:
-        calib = afwImage.Calib(butler.get("calexp_md", vId, immediate=True))
-        calib.setThrowOnNegativeFlux(False)
+        try:
+            calib = afwImage.Calib(butler.get("calexp_md", vId, immediate=True))
+        except TypeError as te:
+            print(te)
+            continue
```

Comment:
Yes, this is from the same problem solved by DM-4133.  Unfortunately problems in reduced data persist even after code has been updated.  I think we'll unfortunately be seeing such issues and accumulating a bit of cruft like this as we have to deal with more and more actual data.

When I ran validateDrp.py on the DECam COSMOS data as processed as NCSA, there were still a number of files with this issue.  This catch is designed to skip those and proceed onward.

MWV

> On Mar 3, 2016, at 10:29, hchiang2 <notifications@github.com> wrote:
> 
> In python/lsst/validate/drp/validate.py:
> 
> > @@ -88,17 +92,23 @@ def loadAndMatchData(repo, visitDataIds,
> >      srcVis = SourceCatalog(newSchema)
> >  
> >      for vId in visitDataIds:
> > -        calib = afwImage.Calib(butler.get("calexp_md", vId, immediate=True))
> > -        calib.setThrowOnNegativeFlux(False)
> > +        try:
> > +            calib = afwImage.Calib(butler.get("calexp_md", vId, immediate=True))
> > +        except TypeError as te:
> > +            print(te)
> > +            continue
> 
> Didn't DM-4133 solve the LTV1/2 mismatched type issue? Or is this from something else?
> 
> —
> Reply to this email directly or view it on GitHub.
>